### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by Nyuhnyash**

### CODE REVIEW

**Review by Dmitry221060 — CHANGES_REQUESTED** [review]
Hello, thank you for your contribution!
Overall translations look good, but I left few suggestions to improve them. You can accept them, propose alternative or comment on why do you think your version would fit better.
Also feel free to add yourself into contributors list in `modinfo.json` ^^

File: assets/rpvoicechat/lang/ru.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "item-handbell": "Колокольчик",
+    "item-royalhorn": "Королевский рожок",
+    "item-royalhornhandle": "Ручка королевского рожка",
+    "item-royalhornhead": "Головка королевского рожка",
+    "item-handheldradio": "Портативное радио",
+    "item-telegraphwire": "Телеграфный провод",
```
[the remark this answers is not in the document]
Comment (Nyuhnyash) [reply assets/rpvoicechat/lang/ru.json:7]: В моде нет телеграфа, значит, и провода от него быть не должно, а "телеграфный" это характеристка                                            провода, не обязательно его принадлежность к телеграфу.

File: assets/rpvoicechat/lang/ru.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "item-radiocase": "Чехол для радио",
+
+    "itemdesc-handbell": "Маленький колокольчик. Серебряное покрытие колокольчика постранному мерцает фиолетовым...",
+
+    "block-callbell": "Вызывной звонок",
```
[the remark this answers is not in the document]
Comment (Nyuhnyash) [reply assets/rpvoicechat/lang/ru.json:12]:
```suggestion
    "block-callbell": "Звонок",
```

File: assets/rpvoicechat/lang/ru.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "Command.Info.Success": "Расстояние шёпота: {0} блоков\nРасстояние разговора: {1} блоков\nРасстояние крика: {2} блоков",
+    "Command.Reset.Desc": "Сбрасывает расстояния для крика, разговора и шёпота к значениям по умолчанию",
+    "Command.Reset.Success": "Расстояния для крика, разговора и шёпота сброшены к значениям по умолчанию",
+    "Hotkey.ModMenu": "RPVoice: Меню мода",
+    "Hotkey.VoiceLevel": "RPVoice: Изменить расстояние голоса",
+    "Hotkey.PTT": "RPVoice: Push to talk",
+    "Hotkey.Mute": "RPVoice: Включение/выключение звука",
+    "Gui.ModMenu.TitleBar": "Меню настроек RP VoiceChat",
+    "Gui.ModMenu.Tab.AudioInput": "Аудиовход",
+    "Gui.ModMenu.Tab.AudioOutput": "Аудиовыход",
```
[the remark this answers is not in the document]
Comment (Nyuhnyash) [reply assets/rpvoicechat/lang/ru.json:32]:
```suggestion
    "Gui.ModMenu.Tab.AudioInput": "Устройство ввода звука",
    "Gui.ModMenu.Tab.AudioOutput": "Устройство вывода звука",
```
or
```suggestion
    "Gui.ModMenu.Tab.AudioInput": "Входное аудиоустройство",
    "Gui.ModMenu.Tab.AudioOutput": "Выходное аудиоустройство",
```

File: assets/rpvoicechat/lang/ru.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    "Gui.ModMenu.Tab.Interface": "Интерфейс",
+    "Gui.ModMenu.Tab.PlayerList": "Список игроков",
+    "Gui.ModMenu.inputDevice.Label": "Устройство ввода",
+    "Gui.ModMenu.inputDevice.Tooltip": "Используемое устройство ввода",
+    "Gui.ModMenu.togglePushToTalk.Label": "Push To Talk",
+    "Gui.ModMenu.togglePushToTalk.Tooltip": "Использовать активацию по нажатию вместо активации по голосу",
+    "Gui.ModMenu.muteMicrophone.Label": "Выключить микрофон",
+    "Gui.ModMenu.muteMicrophone.Tooltip": "Выключить микрофон",
+    "Gui.ModMenu.loopback.Label": "Loopback",
+    "Gui.ModMenu.loopback.Tooltip": "Проигрывать записываемый звук в аудиовыход",
```
[the remark this answers is not in the document]
Comment (Nyuhnyash) [reply assets/rpvoicechat/lang/ru.json:43]: Tooltip is meant to make clear the checkbox purpose.
```suggestion
    "Gui.ModMenu.loopback.Tooltip": "Проигрывать записываемый звук в выходное аудиоустойство",
```

File: assets/rpvoicechat/lang/ru.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Dmitry221060) [reply assets/rpvoicechat/lang/ru.json:7]: "Телеграфный провод" sounds unnatural/incorrect but alright

File: assets/rpvoicechat/lang/ru.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Dmitry221060) [reply assets/rpvoicechat/lang/ru.json:12]: Yeah, sounds good to me. 👍

File: assets/rpvoicechat/lang/ru.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dmitry221060) [reply assets/rpvoicechat/lang/ru.json:32]: AudioInput tab is intended for settings related to recording audio, it is more about audio than about some device
Same with AudioOutput. Also it is preferable that tab names are short and simple for them to look better in the UI and not take too much screen space

File: assets/rpvoicechat/lang/ru.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (Dmitry221060) [reply assets/rpvoicechat/lang/ru.json:43]: I'm fine with either of commented alternatives 👍

**Review by Dmitry221060 — CHANGES_REQUESTED** [review]
Still unhappy with tab names

File: assets/rpvoicechat/lang/ru.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Nyuhnyash) [reply assets/rpvoicechat/lang/ru.json:32]: You're right. I'm sorry, I mistook with other line

File: assets/rpvoicechat/lang/ru.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nyuhnyash) [reply assets/rpvoicechat/lang/ru.json:7]: "[Телеграфный провод](https://context.reverso.net/translation/english-russian/telegraph+wire)" has lots more uses than suggested variant

File: assets/rpvoicechat/lang/ru.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Nyuhnyash) [reply assets/rpvoicechat/lang/ru.json:32]:
```suggestion
    "Gui.ModMenu.Tab.AudioInput": "Входное аудио",
    "Gui.ModMenu.Tab.AudioOutput": "Исходящее аудио",
```

**Review by Dmitry221060 — APPROVED** [review]
Tested in game, looks great 👍 

Merge with squash.

**Pull request merged**